website-components-playground/src/pages/drawer.tsx

```diff
@@ -0,0 +1,81 @@
+import { Drawer } from '@commercetools-frontend/application-components';
+import Spacings from '@commercetools-uikit/spacings';
+import Text from '@commercetools-uikit/text';
+import ModalController from '../components/modal-controller';
+import PlaygroundController from '../components/playground-controller';
+import LayoutApp from '../layouts/layout-app';
+
+const DrawerExample = () => (
+  <LayoutApp>
+    <PlaygroundController
+      knobs={[
+        {
+          kind: 'text',
+          name: 'title',
+          label: 'Title',
+          initialValue: 'Lorem ipsum',
+        },
+        {
+          kind: 'text',
+          name: 'title',
+          label: 'subTitle',
+          initialValue: 'Lorem ipsum dolor sit amet',
+        },
```

Comment:
```suggestion
        {
          kind: 'text',
          name: 'subtitle',
          label: 'Subtitle',
          initialValue: 'Lorem ipsum dolor sit amet',
        },
```

Reply:
updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3513/commits/6591cae5936d0d7665d36a5613ef3c3ac9abfde6
Thank you 🙌🏽